beekeeper-integration-tests/src/test/java/com/expediagroup/beekeeper/integration/BeekeeperExpiredMetadataSchedulerApiaryIntegrationTest.java

```diff
@@ -147,10 +147,12 @@ public void expiredMetadataAddPartitionEvent() throws SQLException, IOException,
         PARTITION_A_VALUES, true);
     amazonSQS.sendMessage(sendMessageRequest(addPartitionSqsMessage.getFormattedString()));
 
-    await().atMost(TIMEOUT, TimeUnit.SECONDS).until(() -> getExpiredMetadataRowCount() == 1);
+    insertExpiredMetadata("s3://location", null);
+
+    await().atMost(30, TimeUnit.SECONDS).until(() -> getExpiredMetadataRowCount() == 2);
 
     List<HousekeepingMetadata> expiredMetadata = getExpiredMetadata();
-    assertExpiredMetadata(expiredMetadata.get(0), LOCATION_A, PARTITION_A_NAME);
+    assertExpiredMetadata(expiredMetadata.get(1), LOCATION_A, PARTITION_A_NAME);
```

Comment:
what happened to 0?

Reply:
I've had to insert some expired metadata - see line 150, which means theres now 2 entries in the table. This test adds a new partition, but because of the max cleanup timestamp stuff I added there needs to be an existing entry for the table.